### PR TITLE
fixes and testing

### LINKED ISSUE #2170: Add synpress plugin for web3 testing in playwright
In order to improve the testing capabilities of our application, we would like to integrate the Synpress plugin for Playwright. This will allow us to perform wallet signing and test authenticated sessions more efficiently, ensuring a more robust and secure application.

## Requirements

- Install and set up the Synpress plugin for Playwright in the project
- Configure the Synpress plugin to work with our current testing environment
- Implement wallet signing functionality to allow testing of blockchain transactions
- Create tests to verify the functionality of authenticated sessions within the application

## Acceptance Criteria

- Synpress plugin is installed and configured in the project
- Wallet signing functionality is implemented and working as expected
- Authenticated session tests are created and passing

## Additional Context

Synpress is a plugin for Playwright that provides an easy way to test Ethereum dApps. It includes features such as wallet signing, key management, and contract interaction. Integrating this plugin will help us streamline the testing process for our blockchain-based application, as well as improve the overall security and robustness of the platform.

https://github.com/drptbl/synpress-examples/blob/master/playwright/isolated-state/tests/connect-wallet.spec.ts

https://github.com/Synthetixio/synpress

### PR DESCRIPTION
**Pull request by Seroxdesign**
## What does this PR do?

Integrate Synpress with Playwright for e2e web3 testing

## Related issues

Fixes #2170

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x ] Enhancement (non-breaking small changes to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Explanation of the changes

Added synpress to testing folders, added tags to buttons so that we can identify and click them in tests

## Emoji

(💯

### CODE REVIEW

File: apps/web/src/components/Shared/Login/WalletSelector.tsx [hunk 1]
```diff
@@ -141,6 +141,7 @@ const WalletSelector: FC<WalletSelectorProps> = ({
       <div className="space-y-2.5">
         {chain?.id === CHAIN_ID ? (
           <Button
+            id="lenster-login"
```
Comment (bigint) [inline apps/web/src/components/Shared/Login/WalletSelector.tsx:144]: we always use `data-testid`

File: tests/playwright.config.ts
Comment (bigint) [inline tests/playwright.config.ts:1]: Lot of things changed in this file, it will break ci 😕

**Review by bigint — CHANGES_REQUESTED** [review]

**Comment by Seroxdesign** [comment]
On it

**Comment by bigint** [comment]
Closing this of being PR got stale for 2 weeks 🙇🏼 😕

**Pull request closed without merging**